diff --git a/node_modules/capnweb/dist/index-workers.js b/node_modules/capnweb/dist/index-workers.js
index ff4527f..f5392b0 100644
--- a/node_modules/capnweb/dist/index-workers.js
+++ b/node_modules/capnweb/dist/index-workers.js
@@ -1384,6 +1384,9 @@ var Evaluator = class _Evaluator {
               return stub;
             }
           };
+          if (!isPromise && value.length == 2 && hook instanceof TargetStubHook) {
+            return hook.getTarget();
+          }
           if (value.length == 2) {
             if (isPromise) {
               return addStub(hook.get([]));
diff --git a/node_modules/capnweb/dist/index.js b/node_modules/capnweb/dist/index.js
index b6f9cdc..e66777b 100644
--- a/node_modules/capnweb/dist/index.js
+++ b/node_modules/capnweb/dist/index.js
@@ -1381,6 +1381,9 @@ var Evaluator = class _Evaluator {
               return stub;
             }
           };
+          if (!isPromise && value.length == 2 && hook instanceof TargetStubHook) {
+            return hook.getTarget();
+          }
           if (value.length == 2) {
             if (isPromise) {
               return addStub(hook.get([]));
